doc/howto/instances_access_files.md

```diff
@@ -0,0 +1,91 @@
+(instances-access-files)=
+# How to access files in an instance
+
+You can access the files from your instance and edit them, delete them, pull them to or push them from the host machine, or mount a file system from the instance to the host machine.
+
+## Edit instance files
+
+To edit an instance file from your host machine, enter the following command:
+
+    lxc file edit <instance_name>/<path_to_file>
+
+For example, to edit the `/etc/hosts` file in the instance, enter the following command:
+
+    lxc file edit my-container/etc/hosts
+
+```{note}
+The file must already exist on the instance.
+You cannot use the `edit` command to create a file on the instance.
+```
+
+## Delete files from the instance
+
+To delete a file from your instance, enter the following command:
+
+    lxc file delete <instance_name>/<path_to_file>
+
+## Pull files from the instance to the host machine
+
+To pull a file from your instance to your host machine, enter the following command:
+
+    lxc file pull <instance_name>/<path_to_file> <file_path_on_host>
+
+For example, to pull the `/etc/hosts` file to the current folder, enter the following command:
+
+    lxc file pull my-instance/etc/hosts .
+
+Instead of pulling the instance file into a file on the host system, you can also pull it to stdin and pipe it into another command.
+This can be useful, for example, to check a log file:
+
+    lxc file pull my-instance/var/log/syslog - | less
+
+To pull a folder with all contents, enter the following command:
+
+    lxc file pull -r <instance_name>/<path_to_folder> <location_on_host>
+
+## Push files from the host machine to the instance
+
+To push a file from your host machine to your instance, enter the following command:
+
+    lxc file push <file_path_on_host> <instance_name>/<path_to_file>
+
+To push a folder with all contents, enter the following command:
+
+    lxc file push -r <location_on_host> <instance_name>/<path_to_folder>
+
+## Mount a file system from the instance
```

Comment:
There's a bit of creativity in this section, so please check if it's technically correct! :wink:

Reply:
Updated as discussed.